browser/ui/views/frame/brave_browser_non_client_frame_view_mac.mm

```diff
@@ -38,3 +39,12 @@
   }
   frame_graphic_->Paint(canvas, bounds_to_frame_graphic);
 }
+
+int BraveBrowserNonClientFrameViewMac::GetTopInset(bool restored) const {
+  if (tabs::features::ShouldShowVerticalTabs()) {
+    // Set minimum top inset to show caption buttons on frame.
+    return 18;
```

Comment:
Ordinarily I'd worry about hard-coding this. I'm not sure if macOS offers to make these buttons larger? I can't find anything. But either way this seems temporary since we'll add title area here or move toolbar up to this area.

Reply:
Yes, I agree. This is a temporary value just for this moment.